definitions.py

```diff
@@ -0,0 +1,21 @@
+"""Module which holds definitions."""
+import os
+
+# paths
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
What does os.path.abspath(__file__) mean ?

Reply:
os.path.abspath(__file__) sets the ROOT_DIR variable with the absolute path to the current working module (returns path/main.py, use dirname to strip down main.py and get only the root directory) 
 __file__ is the constant pathname of the file from which the module was loaded (returns the same, path/main.py); It is absolute except when calling the script directly with using a relative path. I need the absolute path each time, so to make sure it gets it, I added os.path.abspath(__file__) so it returns the absolute path.